pkgs/by-name/or/oreo-cursors-plus/package.nix

```
@@ -0,0 +1,67 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  ruby,
+  inkscape,
+  xorg,
+  cursorsConf ? null, # If set to a string, overwrites contents of './cursors.conf'
+}:
+with lib;
+  stdenv.mkDerivation rec {
```

I think the `echo "${cursorsConf}"` seems a bit easy to break. What happens if there are newlines? How about we create a new file instead, and overwrite the previous file if there's any content?

```suggestion
let
  newCursorsConf = writeText "oreo-cursors-plus.conf" cursorsConf;
in
  stdenv.mkDerivation rec {
```

Then we can use `cp ${newCursorsConf} ./cursors.conf` later